include/ListView/ListViewController.php

```diff
@@ -208,8 +232,11 @@ function getListViewEntries($focus, $module,$result,$navigationInfo,$skipActions
 					if (is_null($field)) continue;
 				}
 				$uitype = $field->getUIType();
-				$rawValue = $this->db->query_result($result, $i, $field->getColumnName());
-				if($module == 'Calendar') {
```

Comment:
I committed the changes. I didn't cancel the row, it was moved at the end of the line that's why you didn't see it, my mistake